### DISCUSSION TITLE
fluent-plugin-sql: is it still maintained?

### DISCUSSION THREAD

**Discussion by rophy** [fluent/fluentd#4419] [General]
Hi, I see the repo is almost 3 years since last human commit - is there still any maintainer for it?

I submitted a pull request to fix crash bug due to incompatibility with ruby > 3.2.0, and hope to get some attention there to have it reviewed:

https://github.com/fluent/fluent-plugin-sql/pull/144

**Comment by ashie** [comment]
Sorry for our less activity on the plugin, but we don't forget it.
Our human resource is very limited, and recently we are busy on addressing some  fatal bugs in Fluntd itself and td-agent/fluent-package.

**Reply by rophy** [reply]
np, thanks for the quick reply!
